rts/lra/coordinator/src/main/resources/META-INF/beans.xml

```diff
@@ -2,5 +2,5 @@
        xsi:schemaLocation="
          http://xmlns.jcp.org/xml/ns/javaee
          http://xmlns.jcp.org/xml/ns/javaee/beans_1_2.xsd"
-       bean-discovery-mode="all">
```

Comment:
Do we need `beans.xml` when CDI for coordinator was discarded?

Reply:
I would keep it, if, for not anything else, Arc in Quarkus won't scan dependencies if there is no beans.xml. The question is whether anything will be exposed through CDI at all.

Reply:
Fair enough.